core/src/main/java/org/sonarsource/sonarlint/core/container/storage/StorageContainer.java

```diff
@@ -138,6 +142,17 @@ public GlobalStorageStatus getGlobalStorageStatus() {
     return getComponentByType(GlobalUpdateStatusReader.class).get();
   }
 
+  public List<LoadedAnalyzer> getAnalyzers() {
+    DefaultPluginRepository pluginRepository = getComponentByType(DefaultPluginRepository.class);
+    return pluginRepository.getPluginInfos().stream()
+      .map(StorageContainer::pluginInfoToAnalyzer)
+      .collect(Collectors.toList());
+  }
+
+  private static LoadedAnalyzer pluginInfoToAnalyzer(PluginInfo p) {
+    return new DefaultLoadedAnalyzer(p.getKey(), p.getName(), p.getVersion() == null ? null : p.getVersion().toString());
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') A "NullPointerException" could be thrown; "getVersion" is nullable here [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS2259)